NEWS.md

```diff
@@ -1,3 +1,16 @@
+# devtools 1.10.0.9001
+
+## New features
```

Comment:
Please don't add headings yourself. I add these during the final release process.